Marlin/dogm_bitmaps.h

```diff
@@ -419,4 +419,145 @@
       0x0C, 0x00, 0x00, 0x06, 0x00, 0x00, 0x06, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
     };
   #endif // Extruders
+
+  #if ENABLED(BABYSTEP_ZPROBE_GFX_OVERLAY)
```

Comment:
These additions have inconsistent spacing as well as missing the binary representation of `cw_bmp`